radix-engine/tests/metering.rs

```diff
@@ -157,7 +157,7 @@ fn test_total_cost_units_consumed() {
     Cost analysis:
     1. Transaction validation cost = TX_VALIDATION_COST_PER_BYTE * 1
     2. Engine run cost
-       * invoke_function: 4515
+       * invoke_function: 4520
```

Comment:
@iamyulong Since you are working on the fees stuff, I wanted to tag you in this to make sure that everything here is correct. 

The fees seem to have increased since our addresses are not 27 bytes long instead of 26. I have updated the number here to reflect that. Does this look good to you based on your calculations?

Reply:
That's understandable. Part of invoke_function cost comes from payload size. The delta `10` seems to match the address size change.